index.js

```diff
@@ -48,10 +35,13 @@ class BullFacility extends Base {
   _stop (cb) {
     async.series([
       next => { super._stop(next) },
-      next => {
+      async next => {
```

Comment:
2 questions
1. why do you use callback with async method? Additionally, you're not catching possible error from async function exectution.
2. does it work fine in async when you mix both cb-style functions and async functions?